supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchJobDefinition.scala

```diff
@@ -68,22 +70,47 @@ trait AwsBatchJobDefinitionBuilder {
   def builder(commandLine: String, dockerImage: String): ContainerProperties.Builder =
     ContainerProperties.builder().command("/bin/bash", "-c", commandLine).image(dockerImage)
 
-  def buildResources(builder: ContainerProperties.Builder, runtimeAttributes: AwsBatchRuntimeAttributes, taskId: String): ContainerProperties.Builder = {
+  def buildResources(builder: ContainerProperties.Builder,
+                     runtimeAttributes: AwsBatchRuntimeAttributes,
+                     uniquePath: String,
+                     rcPath: String,
+                     jobDescriptor: BackendJobDescriptor,
+                     callExecutionRoot: Path): ContainerProperties.Builder = {
+    // The initial buffer should only contain one item - the hostpath of the
+    // local disk mount point, which will be needed by the docker container
+    // that copies data around
+    val environment =
+      runtimeAttributes.disks.collect{
+        case d if d.name == "local-disk" =>
+          KeyValuePair.builder.name("AWS_CROMWELL_LOCAL_DISK").value(d.mountPoint.toString).build
+      }.toBuffer
+    environment.append(KeyValuePair.builder.name("AWS_CROMWELL_PATH").value(uniquePath).build)
+    environment.append(KeyValuePair.builder.name("AWS_CROMWELL_RC_FILE").value(rcPath).build)
+    environment.append(KeyValuePair.builder.name("AWS_CROMWELL_CALL_ROOT").value(callExecutionRoot.toString).build)
+    // We want a marker for the process monitor to know for certain this container is from
+    // Cromwell. Rather than relying on some of the other environment variables, we're using
+    // a unique marker variable that doesn't mean much to anything else (and therefore has
+    // no reason to be passed on. Also, by abbreviating CROMWELL unlike the other variables,
+    // any process scanning for all "AWS_CROMWELL" will skip by this particular one.
```

Comment:
TOL a name that describes what this variable is actually for would be preferable, e.g. `AWS_BATCH_PROCESS_MONITOR_MARKER` or something like that.

Reply:
Sure - I can make it clearer. I'll go with AWS_BATCH_CROMWELL_PROCESS_MONITOR_MARKER